classes/app.rb

```diff
@@ -0,0 +1,167 @@
+require_relative 'person'
+require_relative 'teacher'
+require_relative 'student'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'classroom'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_all_books
+    puts 'List of all books:'
+    @books.each { |book| puts "#{book.title} by #{book.author}" }
+  end
+
+  def list_all_people
+    puts 'List of all people:'
+    @people.each.with_index(1) do |person, index|
+      person_type = person.is_a?(Teacher) ? 'Teacher' : 'Student'
+      puts "#{index}. #{person.name} (#{person_type})"
+    end
+  end
+
+  def list_all_people_with_indexes
+    puts 'List of all people:'
+    @people.each_with_index do |person, index|
+      person_type = person.is_a?(Teacher) ? 'Teacher' : 'Student'
+      puts "#{index + 1}. #{person.name} (#{person_type})"
+    end
+  end
```

Comment:
- [Optional] Great job for fixing the rental listing issue, I think it would be better if you list the people with their `id` to make it easy for the rental list 👍